### PR TITLE
Rewrite Windows building instructions/dependencies README

### PR DESCRIPTION
**Pull request by mfwass**
Rewritten windows building instructions and dependencies readme

### CODE REVIEW

**Comment by bobbybee** [comment]
The build instructions are very nice, thank you. However, the README in the dependencies folder should be platform-independent instructions (for users who choose not to follow a particular guide for the system, or who are compiling for systems that are not officially supported). Could you rework the dependencies README to point Windows users to your guide while still maintaining platform neutral instructions, for, say, Linux users?

As well, the dependencies README is not designed to be a detailed instruction set (as opposed to `windows-visualstudio.md`). Please keep it concise :)

**Comment by bobbybee** [comment]
:+1: 

Once someone else reviews this, you're good to go :)

**Comment by mfwass** [comment]
Okie. :)

**Comment by AlJaMa** [comment]
A better place for this would be our git.io repo where are docs live

**Comment by mfwass** [comment]
Would that require it to be converted from markdown to HTML?

**Comment by xHarv14** [comment]
@mfwass If you put it in the raw source (not astron.github.io) then you should be able to dump it in as a raw markup file. All of the existing markup files are here: https://github.com/Astron/astron.docs/tree/master/github.io/docs/astron/en/master

**Comment by mfwass** [comment]
@xHarv14 I have put it in the folder https://github.com/Astron/astron.docs/tree/master/github.io/docs/astron/en/master and have made a pull request.

File: dependencies/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Dependencies
+============
+
+1. YAML-CPP
+-----------
+Downloadable here: http://code.google.com/p/yaml-cpp/downloads/.
```
Comment (AlJaMa) [inline dependencies/README.md]: This link is out of date.

File: dependencies/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Download version 0.5.1 or newer.
+
+2. OpenSSL
+----------
+Windows version downloadable here: http://www.npcglib.org/~stathis/blog/precompiled-openssl/.
```
Comment (AlJaMa) [inline dependencies/README.md]: Do not lead users to precompiled OpenSSL

File: dependencies/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+For other operating systems, use your local package manager to install and compile OpenSSL. For example: `brew install OpenSSL`
+
+Download the latest the precompiled version. Openssl-1.0.1j-vs2013.7z at the time of this writing.
+
+3. Boost
+--------
+Downloadable here: http://www.boost.org/users/download/.
+
+For other operating systems, use your local package manager to install and compile Boost. For example: `brew install boost`
```
Comment (AlJaMa) [inline dependencies/README.md]: Specify that `brew install boost` is OS X.

File: docs/building/windows-visualstudio.md [hunk 4]
```diff
@@ -1,31 +1,242 @@
...
-dependencies.  See the [building readme](build-readme.md) for instructions.
+Dean Aug ~ (1/3/2015)
 
+Michael "mwass" Wass ~ (6/15/2015)
 
-### Preparing the VS Project ###
-_Currently, Astron only supports Visual Studio 2013 and above._
+1. Introduction
+---------------
+This document describes all steps required to build the *Astron* cluster on a windows machine; from acquisition of the *Astron* code and all code components it depends on (referred to as "dependencies"), to building of those dependencies and *Astron* itself. These are general instructions that can be followed by anyone using *Astron*.
```
Comment (AlJaMa) [inline docs/building/windows-visualstudio.md]: remove "cluster" you are really just building the program.

File: docs/building/windows-visualstudio.md [hunk 5]
```diff
@@ -1,31 +1,242 @@
...
+`openssl 1.0.1+`
+
+Each must be downloaded and installed in the *Astron* source tree created in the previous section.
+
+These are the dependencies for Astron compiled for use with a YAML Database.
+
+### 1. YAML-CPP
+The available *yaml-cpp* downloads are listed on the following site:
+
+`https://code.google.com/p/yaml-cpp/downloads/list`
```
Comment (AlJaMa) [inline docs/building/windows-visualstudio.md]: Out of date

File: docs/building/windows-visualstudio.md [hunk 6]
```diff
@@ -1,31 +1,242 @@
...
+
+From the site listed above, select the link at the top to download the latest version of *yaml-cpp* (version 0.5.1 at the time of this writing).
```
Comment (AlJaMa) [inline docs/building/windows-visualstudio.md]: out of date

File: docs/building/windows-visualstudio.md [hunk 7]
```diff
@@ -1,31 +1,242 @@
...
+
+`Compiling-Astron\Astron\dependencies`
+
+This will create a *boost* subdirectory (named after the version), containing all the files for that version.
+
+### 3. OpenSSL
+
+The available *OpenSSL* downloads are listed on the following site:
+
+`http://www.npcglib.org/~stathis/blog/precompiled-openssl`
```
Comment (AlJaMa) [inline docs/building/windows-visualstudio.md]: Do not link directly to precompiled OpenSSL

File: dependencies/README.md (hunk 2 above)
Comment (mfwass) [inline dependencies/README.md]: What do you suggest we do in place?

**Pull request closed without merging**